README.md

```diff
@@ -127,6 +127,38 @@ Custom registries are not currently supported
 ([#160](https://github.com/obi1kenobi/cargo-semver-checks/issues/160)), so crates published on
 registries other than crates.io should use one of the other approaches of generating the baseline.
 
+#### Git repository detection
+
+When looking up a git revision with `--baseline-rev`,
+`cargo-semver-checks` will walk up the current directory until it finds a `.git/` directory to resolve the revision and extract the corresponding worktree.
+You can use the following environment variables to influence the git repository detection:
+
+- [`GIT_CEILING_DIRECTORIES`]: a colon (`:`) separated list of absolute paths which `cargo-semver-checks` should not search for the `.git/` directory.
+  This can be used to prevent `cargo-semver-checks` from searching slow network mounted directories.
+  This environment variable cannot be used to prevent searching the current working directory,
+  or a directory explicitly set with [`GIT_DIR`].
+  If this is not set,
+  `cargo-semver-checks` will search the current working directory and any of its parents (up to the root directory).
+- [`GIT_DIR`]: explicitly set the location of the `.git/` directory
+  (if the value is a relative path, it is resolved from the current working directory).
+  If this is not set,
+  `cargo-semver-checks` will search the current working directory and any of its parents for the `.git/` directory.
```

Comment:
`GIT_DIR` seems like the option we'd expect to be most commonly used, right?

If so, can we put it in the top spot? It feels like describing `GIT_CEILING_DIRECTORIES` will flow better in the second spot anyway, because if `GIT_DIR` isn't set it's presumably _then_ that we kick off a search based on parameters that `GIT_CEILING_DIRECTORIES` influences. If so, let's document that setting `GIT_CEILING_DIRECTORIES` has no effect if `GIT_DIR` is set too.